ci/images/plugin-compiler/data/build.sh

```diff
@@ -7,16 +7,14 @@ plugin_id=$2
 # GOOS and GOARCH can be send to override the name of the plugin
 GOOS=$3
 GOARCH=$4
-CGOENABLED=0
-
-  PLUGIN_BUILD_PATH="/go/src/plugin_${plugin_name%.*}$plugin_id"
 
 function usage() {
     cat <<EOF
 To build a plugin:
-      $0 <plugin_name> <plugin_id>
+      $0 <plugin_name> [<plugin_id>] [<GOOS>] [<GOARCH>]
```

Comment:
Rather than taking GOOS and GOARCH as command line vars, it would be clearer to the plugin author if we just inherited from the env. It can be set by `docker -e GOOS=arm64 ...` which makes it clear that the plugin is being built via the go cross compiler toolchain and the nuances that apply when using modules with C dependencies.

Reply:
Yes, that makes sense. It will be more consistent. Will update the implementation.